soh/src/overlays/actors/ovl_Item_B_Heart/z_item_b_heart.c

```diff
@@ -99,8 +99,10 @@ void ItemBHeart_Draw(Actor* thisx, GlobalContext* globalCtx) {
     }
 
     if (gSaveContext.n64ddFlag) {
-        GetItem_Draw(globalCtx,
-                     Randomizer_GetItemModelFromId(Randomizer_GetRandomizedItemId(GI_HEART_CONTAINER_2, this->actor.id, this->actor.params, globalCtx->sceneNum)));
+        GetItem_Draw(globalCtx, ABS(Randomizer_GetRandomizedItem(GI_HEART_CONTAINER_2, this->actor.id,
+                                                                 this->actor.params, globalCtx->sceneNum)
+                                        .gi) -
+                                    1);
```

Comment:
some funky formatting here

Reply:
Yeah VSCode was doing it automatically. I guess I could probably just turn autoformat off for C/C++ files?

Reply:
yeah, autoformat defaults can be funky sometimes, might be worth keeping it off until we have some sort of linting set up

i think the plan is to add linting to the ci

@louist103 and i were discussing linting the other day, the idea is to wait for a good time (like when we stop dev on rachael and start doing everything in zhora maybe) to add linting, then do a giant PR formatting everything, then enforce linting via ci for all PRs after that

Reply:
Addressed by 4cc492f383094550d89b12c9759c83130d91f1e6, as well another spot or two that had similar weird formatting.